abr_control/controllers/path_planners/s_shaped_dmp.py

```diff
@@ -0,0 +1,42 @@
+"""
+Running operational space control with a PyGame display, and using the pydmps
+library to specify a trajectory for the end-effector to follow, in
+this case, a circle.
+
+To install the pydmps library, clone https://github.com/studywolf/pydmps
+and run 'python setup.py develop'
+"""
+import numpy as np
+
+try:
+    import pydmps
+except ImportError:
+    print('\npydmps library required, see ' +
+          'https://github.com/studywolf/pydmps\n')
+
+import scipy.interpolate
+from .dmp_filter import dmpFilter
+
+
+class SShapedDmp(dmpFilter):
```

Comment:
is it worth keeping the s shaped dmp path planner?

Reply:
in dewolf2018